.gitignore

```diff
@@ -115,19 +115,8 @@ public/stylesheets/jmaki-standard-right-sidebar.css
 # tmp/
 tmp/*
 
-# vendor/bundle
```

Comment:
@himdel Can you leave this in? There is some convention around having this for anyone who vendorizes their gems: https://github.com/github/gitignore/blob/master/Ruby.gitignore#L37